tsfc/logging.py

```diff
@@ -0,0 +1,11 @@
+"""Logging for TSFC."""
+
+from __future__ import absolute_import
+
+import logging
+
+logger = logging.getLogger('tsfc')
+logger.setLevel(logging.INFO)
+
+default_handler = logging.StreamHandler()
```

Comment:
Yes, this can be done like so:
```python
from firedrake import *

import logging

logger = logging.getLogger("tsfc")

if op2.MPI.comm.rank == 0:
    ch = logging.StreamHandler()

    ch.setFormatter(logging.Formatter("Banananas %(message)s"))
else:
    ch = logging.NullHandler()

for hdlr in logger.handlers:
    logger.removeHandler(hdlr)
logger.addHandler(ch)
```